src/jit/lower.cpp

```diff
@@ -3118,13 +3118,17 @@ GenTree* Lowering::LowerVirtualStubCall(GenTreeCall* call)
 
         // Direct stub calls, though the stubAddr itself may still need to be
         // accesed via an indirection.        
-        GenTree* tmp = AddrGen(stubAddr);
-        tmp->gtRegNum = REG_VIRTUAL_STUB_PARAM;
-
-        tmp = Ind(tmp);
-        tmp->gtRegNum = REG_JUMP_THUNK_PARAM;
-
-        result = tmp;
+        GenTree* addr = AddrGen(stubAddr);
+        GenTree* indir = Ind(addr);
```

Comment:
This logic depends on Indir(addr) is marked as contained by lower and codegen will generate call dword ptr [rel32].  Should we assert this in codegenxarch.cpp?